Fix!: remove parachain id arg (#492)
interlay/interbtc: docs/rococo.md

> # Rococo - Local (Automated Setup, recommended)

Reviewer: ```suggestion
# Rococo - Local (Automated Setup, Recommended)
```

sudo cp ./target/release/interbtc-parachain /usr/local/bin/
```

> Run polkadot-launch. We provide two configations: one to start a [single parachain](./rococo-local-single-chain.json), and one that sets up [two parachains](rococo-local-xcm-setup.json) that can be used for testing xcm communication.

Reviewer: ```suggestion
Run polkadot-launch. We provide two configurations: one to start a [single parachain](./rococo-local.json), and one that sets up [two parachains](rococo-local-xcm.json) that can be used for testing xcm communication.
```

Reviewer: @sander2 these now do not match the names of the configuration files. Feel free to revert this actually or rename the files more succinctly.

```shell
polkadot-launch ./docs/rococo-local-xcm-setup.json
# or..
polkadot-launch ./docs/rococo-local-single-chain.json
```

> # Rococo - Local (Manual Setup, not recommended)

Reviewer: ```suggestion
# Rococo - Local (Manual Setup, Not Recommended)
```
